spec/amber/amber_spec.cr

```diff
@@ -9,15 +9,15 @@ describe Amber::Server do
         server.env = "development"
         server.log = ::Logger.new(STDOUT)
         server.log.level = ::Logger::INFO
-        server.secret = "some secret key"
+        server.secret_key_base = "some secret key"
```

Comment:
Rails uses secret_key_base and then bases other keys off of it. So that they'll stay the same every-time the project runs but every key in the system can still be different because they use secret_key_base and as seed.